apps/mobile/src/features/send/navigation.tsx

```diff
@@ -82,7 +82,7 @@ function getInitialRouteName({
   return 'form';
 }
 
-export function cardStyleInterpolator({
+function cardStyleInterpolator({
```

Comment:
Maybe we can refactor this? Checking the usage and it's only used once so it could be easier to pass inline?

```
 <SendStack.Navigator
      initialRouteName={initialRouteName}
      screenOptions={{
        headerShown: false,
        cardStyle: { backgroundColor: theme.colors['ink.background-primary'] },
        cardStyle: {
      opacity: current.progress,
    },,
      }}
    >
```